LeaderboardBackend/Services/Impl/UserService.cs

```diff
@@ -16,7 +16,22 @@ public UserService(ApplicationContext applicationContext, IConfiguration config)
 
 	public async Task<User?> GetUserById(Guid id)
 	{
-		return await _applicationContext.Users.FindAsync(id);
+		User? user = await _applicationContext.Users.FindAsync(id);
+		if (user is null)
+		{
+			return null;
+		}
+
+		try
+		{
+			user.Modships = await _applicationContext.Modships
```

Comment:
I think it's kinda weird to have some methods in this service that load additional properties for Users and some that don't. I think loading the modships makes more sense in the ModshipService.